apps/aot/src/app/events/[year]/[day]/_components/description/index.tsx

```diff
@@ -89,7 +89,7 @@ export function Description({ challenge }: Props) {
                     'rounded-full',
                   )}
                 >
-                  <Share className="h-4 w-4" />
+                  <Share aria-label="Share" className="h-4 w-4" />
```

Comment:
These elements don't provide their own label or title themselves. So there's nothing in them by default that says what they are in HTML.